src/api/app/views/webui/users/token_triggers/_breadcrumb_items.html.haml

```diff
@@ -2,5 +2,7 @@
   = link_to('Your Profile', user_path(User.session!))
 
 - if action_name == 'show'
+  %li.breadcrumb-item{ 'aria-current' => 'page' }
```

Comment:
```suggestion
  %li.breadcrumb-item
```

This isn't the current page, so setting `aria-current` is wrong here.

I've noticed that this is also wrong in the breadcrumbs from `src/api/app/views/webui/users/tokens/_breadcrumb_items.html.haml`. This could also be fixed in this PR: 

https://github.com/openSUSE/open-build-service/blob/528a66a6ee5e55b497f561f092e837fbc05abc07/src/api/app/views/webui/users/tokens/_breadcrumb_items.html.haml#L9-L10

https://github.com/openSUSE/open-build-service/blob/528a66a6ee5e55b497f561f092e837fbc05abc07/src/api/app/views/webui/users/tokens/_breadcrumb_items.html.haml#L14-L15

Reply:
True. Fixed. Thanks!